src/software/ai/hl/stp/tactic/crease_defender/simulated_crease_defender_tactic_test.cpp

```diff
@@ -83,8 +83,7 @@ TEST_F(SimulatedCreaseDefenderTacticTest, test_not_bumping_ball_towards_net)
     tactic->updateControlParams(enemy_threat_point, alignment);
     setTactic(tactic);
     setRobotId(0);
-    setMotionConstraints({MotionConstraint::ENEMY_ROBOTS_COLLISION,
-                          MotionConstraint::FRIENDLY_DEFENSE_AREA});
+    setMotionConstraints({MotionConstraint::FRIENDLY_DEFENSE_AREA});
```

Comment:
whyd we remove the enemy robots collision???????????????????????????????????????????//

Reply:
ENEMY_ROBOT_COLLISION is not a thing, cuz we want to collide, but gently and with loving care